packages/commands/src/utils.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  sdk,
+  SolanaRelay,
+  Configuration,
+  createHedgehogWalletClient,
+  type AudiusSdk,
+  encodeHashId,
+  decodeHashId
+} from '@audius/sdk'
+
+import {
+  Hedgehog,
+  WalletManager,
+  getPlatformCreateKey,
+  type GetFn,
+  type SetAuthFn,
+  type SetUserFn
+} from '@audius/hedgehog'
+
+import { PublicKey } from '@solana/web3.js'
+
+import { LocalStorage } from 'node-localstorage'
+const localStorage = new LocalStorage('./local-storage')
+
+export const parseUserId = async (arg: string) => {
+  if (arg.startsWith('@')) {
+    // @handle
+    const audiusSdk = await initializeAudiusSdk()
+    const { data } = await audiusSdk.users.getUserByHandle({
+      handle: arg.slice(1)
+    })
+    const id = data!.id
+    return decodeHashId(id)!
+  } else if (arg.startsWith('#')) {
+    // #userId
+    return Number(arg.slice(1))
+  } else {
+    // encoded user id
+    return decodeHashId(arg)!
+  }
+}
+
+let audiusSdk: AudiusSdk | undefined
+let currentUserId: number | undefined
+let currentHandle: string | undefined
+
+let hedgehog: Hedgehog | undefined
+export const getHedgehog = () => {
+  const getFn: GetFn = async (args) => {
+    const res = await fetch(
+      `http://audius-protocol-identity-service-1/authentication?lookupKey=${args.lookupKey}`,
```

Comment:
nit: you could pull out `http://audius-protocol-identity-service-1` to a constant
buttttt idc